### PR TITLE
Feature/1070 network partials

### PR DESCRIPTION
**Pull request by SamuelMeilleur**
# Description

**Make sure the PR is against the `develop` branch (see [Contributing](https://github.com/nccgroup/ScoutSuite/blob/master/CONTRIBUTING.md)).**

Please include a summary of the change(s) and which issue(s) it addresses. Please also include relevant motivation and context.

Fixes # (issue)

## Type of change

Select the relevant option(s):

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (optional)
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: ScoutSuite/frontend/src/components/ServiceCard/index.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
```
Comment (SamuelMeilleur) [inline ScoutSuite/frontend/src/components/ServiceCard/index.js:4]: i removed all uses of FontAwesome icons, if it's ok with you, we'll be able to remove the library from our package.json

File: ScoutSuite/frontend/src/components/LazyPartial/index.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import { useAPI } from '../../api/useAPI';
 import './style.scss';
 
 const propTypes = {
-  data: PropTypes.objectOf({
+  data: PropTypes.shape({
```
Comment (loic294) [inline ScoutSuite/frontend/src/components/LazyPartial/index.js:9]: Ah nice! Je me demandais pourquoi j'avais toujours une erreur...

File: ScoutSuite/frontend/src/components/ServiceCard/index.js (hunk 1 above)
Comment (loic294) [reply ScoutSuite/frontend/src/components/ServiceCard/index.js:4]: Yep! Good with me

**Review by loic294 — APPROVED** [review]
Good!

**Pull request merged**